compass/ocean/tests/parabolic_bowl/viz/__init__.py

```diff
@@ -233,7 +239,7 @@ def rmse_plots(self):
         for i in range(len(self.resolutions) - 1):
             rmse_1st_order[i + 1] = rmse_1st_order[i] / 2.0
 
-        ax.loglog(self.resolutions, np.flip(rmse_1st_order),
+        ax.loglog(self.resolutions, rmse_1st_order,
```

Comment:
@cbegeman, I forgot I made this change in this PR related to #765 .

Reply:
@sbrus89 Great. I just closed the issue.